client/src/client/database_manager_client.py

```diff
@@ -183,15 +183,15 @@ def save_message(
                     id=message.id,
                     body=message.body,
                     date=message.date,
-                    state=state,
+                    state=state.value,
```

Comment:
Why did the old way suddenly stop working? It was much cleaner

Reply:
It never worked. The messages have default value set to delivered in the model and that's why writes didn't fail even when I passed wrong type of data.